OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1149,6 +1149,51 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				if (engineVersion < 20150603)
```

Comment:
It's always nice to leave a comment on each upgrade rule to say what it's about. Makes navigating this huge file much easier.